### PR TITLE
WRP-6287: Increase automated test for Date/TimePicker.

### PR DESCRIPTION
**Pull request by dongsuwon**
Enact-DCO-1.0-Signed-off-by: Dongsu Won (dongsu.won@lgepartner.com)

### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
- Need to add ui tests for DatePicker.
- Need to add ui tests for TimePicker.
- Need to add ss tests for DatePicker.
- Need to add ss tests for TimePicker.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Add ui tests for DatePicker.
- Add ui tests for TimePicker.
- Add ss tests for DatePicker.
- Add ss tests for TimePicker.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]:# (Related issues, references)
WRP-6287

### CODE REVIEW

File: tests/ui/specs/TimePicker/TimePicker-specs.js [hunk 1]
```diff
@@ -93,6 +93,59 @@ describe('TimePicker', function () {
...
+					expect((await extractValues(timePicker)).minute).to.equal(expected);
+					// Step 5: Press 5-way Down on the minute picker.
+					await Page.spotlightDown();
+					// Step 5-1 Verify: Spotlight on the minute picker.
+					// Step 5-2 Verify: The minute value is changed downward.
+					expect(await timePicker.minute.isFocused()).to.equal(true);
+					expect((await extractValues(timePicker)).minute).to.equal(minute);
+				});
+
+				it('should move to the next picker with 5-way OK key [QWTC-2549]', async function () {
```
Comment (juwonjeong) [inline tests/ui/specs/TimePicker/TimePicker-specs.js:126]: I think this case is for QWTC-2541.

File: tests/screenshot/apps/components/TimePicker.js [hunk 2]
```diff
@@ -20,6 +20,27 @@ const TimePickerTests = [
...
+	...withConfig({locale: 'zh-HK'}, [
+		<TimePicker defaultValue={new Date(2009, 5, 7, 11)} />,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 13)} />
+	]),
+	...withConfig({locale: 'am-ET'}, [
+		<TimePicker defaultValue={new Date(2009, 5, 7, 9)}>AM</TimePicker>,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 12)}>Noon</TimePicker>,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 13)}>PM</TimePicker>,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 18)}>Evening</TimePicker>,
+		<TimePicker defaultValue={new Date(2009, 5, 7, 24)}>Late Evening</TimePicker>
```
Comment (juwonjeong) [inline tests/screenshot/apps/components/TimePicker.js:41]: Do we need children prop for these Timepicker? Could you please check it?

File: tests/ui/specs/TimePicker/TimePicker-specs.js [hunk 3]
```diff
@@ -237,6 +290,61 @@ describe('TimePicker', function () {
...
+	});
+
+	describe('es-ES locale', function () {
+		const timePicker = Page.components.timePickerDefault;
+
+		beforeEach(async function () {
+			await Page.open('?locale=es-ES');
+		});
+
+		it('should move to the next picker with 5-way OK key in es-ES locale', async function () {
```
Comment (juwonjeong) [inline tests/ui/specs/TimePicker/TimePicker-specs.js:327]: Would you please add [QWTC-2541] in this description?
```suggestion
		it('should move to the next picker with 5-way OK key in es-ES locale [QWTC-2541]', async function () {
```

File: tests/screenshot/apps/components/TimePicker.js (hunk 2 above)
Comment (dongsuwon) [reply tests/screenshot/apps/components/TimePicker.js:41]: In Amharic, the time notation is different from the standard. Even though we input standard time, test result is different.
e.g)Standard noon: 12:00, Amharic noon: 06:00
It is to check the test result difference in Amharic.

File: tests/ui/specs/TimePicker/TimePicker-specs.js (hunk 1 above)
Comment (dongsuwon) [reply tests/ui/specs/TimePicker/TimePicker-specs.js]: It's my mistake. I fixed it.

File: tests/ui/specs/TimePicker/TimePicker-specs.js (hunk 3 above)
Comment (dongsuwon) [reply tests/ui/specs/TimePicker/TimePicker-specs.js]: Suggested Changes.

File: tests/screenshot/apps/components/TimePicker.js (hunk 2 above)
Comment (juwonjeong) [reply tests/screenshot/apps/components/TimePicker.js:41]: Okay.

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**